skills/lark-sheets/references/lark-sheets-write-cells.md

```
@@ -218,14 +241,14 @@ lark-cli sheets +dropdown-set \
   --options '["待开始","进行中","已完成","已取消"]'
 ```
 
-**`--options` 模式 — 指定颜色**（4 个选项配 3 个颜色，第 4 个按色板补）：
+**`--options` 模式 — 明确的状态语义配色**（仅在用户要求或颜色有助于理解时使用）：
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)